web/src/user/settings/UserSettingsSidebar.tsx

```diff
@@ -55,7 +56,7 @@ export const UserSettingsSidebar: React.FunctionComponent<UserSettingsSidebarPro
                 <SidebarGroupItems>
                     {props.items.account.map(
                         ({ label, to, exact, condition = () => true }) =>
-                            condition({ user: props.user }) && (
+                            condition({ user: props.user, authenticatedUser: props.authenticatedUser }) && (
```

Comment:
```suggestion
                            condition({ user: props.user, authenticatedUser: props.authenticatedUser! }) && (
```

the map loop is a inner function so it doesn't know if the value of authenticated user changed since the non null check on l38

Reply:
A better solution would be to define
```ts
const context = {
    user: props.user,
    authenticatedUser: props.authenticatedUser,
}
```
after the `if` check and using `context` here
```suggestion
                            condition(context) && (
```

Reply:
@eseliger It is actually forbidden by ESLint that trying to make a nullable object to be not null. Please see my comment here: https://github.com/sourcegraph/sourcegraph/pull/10473/files#r422494624

Reply:
@unknwon it's only warned against because it degrades type safety (but it won't fail CI). But my suggestion above would avoid it :)

Reply:
@felixfbecker Magic! Looks like worked, PTAL! :) https://github.com/sourcegraph/sourcegraph/pull/10473/commits/4b3cc81da84fb5b902d45a095d61445d381c7876